electrum/gui/qml/components/TxDetails.qml

```diff
@@ -362,7 +362,7 @@ Pane {
                 onClicked: {
                     if (txdetails.shouldConfirm) {
                         var dialog = app.messageDialog.createObject(app, {
-                            text: qsTr('Confirm signing non-standard transaction?'),
+                            text: qsTr('Confirm signing transaction despite warnings?'),
```

Comment:
note: I've changed this phrasing as "(non-)standard transaction" has a specific meaning in bitcoind (re relay policy) and hence I found it confusing. Though most users probably don't know that meaning. Still, I think it is better not to overload the term.